BUILD.bazel

```diff
@@ -16,6 +16,29 @@ package(default_visibility = [
     "//visibility:private",
 ])
 
+proto_library(
+    name = "stats_proto",
+    srcs = ["grpc/core/stats.proto"],
+)
+
+proto_library(
+    name = "benchmark_service_proto",
+    srcs = [
+        "grpc/testing/benchmark_service.proto",
+        "grpc/testing/messages.proto",
+    ],
+)
+
+proto_library(
+    name = "worker_service_proto",
+    srcs = ["grpc/testing/worker_service.proto",
+            "grpc/testing/control.proto",
```

Comment:
This also doesn't look like it should be included, as it will be used by report_qps_scenario_service.proto.

Reply:
Done